infra/server.py

```diff
@@ -221,6 +221,8 @@ def _sort_tasks_by_duration(self, tasks):
   @cherrypy.tools.no_caching()
   def job_status(self, job_id):
     tasks = self.results_store.fetch_task_rows_for_job(job_id)
+    if len(tasks) == 0:
+      return "No tasks found for specified job_id %s" % job_id
```

Comment:
Shouldn't this still be returning a JSON struct with the expected fields, instead of a non-JSON string?

Reply:
Good catch, I looked at the wrong example. :)